tests/ResponseTest.php

```
@@ -111,6 +112,64 @@ public function testSetContent()
         $this->assertSame('Hello World', (string) $response->getBody());
     }
 
+    public function testSetContentWithResource()
+    {
+        $response = new Response();
+
+        $this->assertSame($response, $response->setContent(fopen('php://input', 'rb')));
+        $this->assertInstanceOf(MessageBody::class, $response->getBody());
+        $this->assertSame('', (string) $response->getBody());
+    }
+
+    public function testIsStatusCode()
+    {
+        $response = new Response();
+
+        $this->assertTrue($response->isStatusCode(200));
+        $this->assertFalse($response->isStatusCode(400));
+    }
+
+    /**
+     * @expectedException        \RuntimeException
+     * @expectedExceptionMessage Could not parse response (error 1).
+     */
+    public function testParseShouldReturnRuntimeExceptionError1()
```

Please rename to Throw instead of Return, same for the 2 methods below.